src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -246,3 +246,14 @@ export const renderWidgetDowntimeNotification = (appName, sectionTitle) => (
   }
   return children;
 };
+
+// receiving formatted date strings in the response
+// so we need to convert back to moment before sorting
```

Comment:
this is a nit, but you mention moment in the comment, but its been removed as a dependency

Reply:
good point. Will remove it.